5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
> OPTx arguments are mess, I agree, though I have no idea how to fix them,,

Just a stupid idea I've been thinking about for only few minutes, so it could be, well, stupid. Can't you just parse the label for the occurrence of $UNKNOWN variable, let the user use "atomic install --unknown=something" or "atomic install --set unknown=something" or something similar to "-e UNKNOWN=something" and replace $UNKNOWN with "something" the same way as you do for OPTx?

That way, the creator of docker image could at least decide the names of the variables.